cns/restserver/api.go

```diff
@@ -716,8 +716,8 @@ func (service *HTTPRestService) getHealthReport(w http.ResponseWriter, r *http.R
 	logger.Response(service.Name, resp, resp.ReturnCode, err)
 }
 
-func (service *HTTPRestService) setOrchestratorType(w http.ResponseWriter, r *http.Request) {
-	logger.Printf("[Azure CNS] setOrchestratorType")
+func (service *HTTPRestService) SetOrchestratorType(w http.ResponseWriter, r *http.Request) {
```

Comment:
is there any reason to change this to exported API?

Reply:
Evan and I determined that we put localhost handlers into different file restserver/v2/ where is for this echo server specifically; and this local echo server is to start as localhost, so we need to expose this API from api.go.